indico/core/db/sqlalchemy/util/models.py

```diff
@@ -212,10 +212,11 @@ def populate_from_dict(self, data, keys=None, skip=None):
             if not hasattr(cls, key):
                 raise ValueError("{} has no attribute '{}'".format(cls.__name__, key))
             old_value = getattr(self, key, None)
-            if old_value != value:
-                # XXX: we copy because of https://bitbucket.org/zzzeek/sqlalchemy/issues/3913/
-                changed[key] = (copy(old_value), value)
-                setattr(self, key, value)
+            setattr(self, key, value)
+            new_value = getattr(self, key, None)
```

Comment:
i don't think you need the `, None` here - after the setattr the attribute should always exist